app/templates/gulpfile.js

```diff
@@ -670,7 +734,12 @@ function startTests(singleRun, done) {
       child.kill();
     }
     if (karmaResult === 1) {
-      done('karma: tests failed with code ' + karmaResult);
+      if (!!args.ignoreErrors) {
+        log($.util.colors.red('karma: tests failed with code ' + karmaResult));
+        done();
+      } else {
+        done(new Error('karma: tests failed with code ' + karmaResult));
+      }
```

Comment:
Using new Error here prevents the need to wrap done callback in an anonymous function earlier, just to prevent the string message to get passed through, while the callback expects null or an Error object.

Allowed for ignoring errors here, e.g. report them, but complete gulp build anyhow..